src/runtime/api-channel.ts

```diff
@@ -121,6 +122,10 @@ function RedundantInitializer(target: {}, propertyKey: string, parameterIndex: n
   set(target.constructor, propertyKey, parameterIndex, {type: MappingType.Direct, initializer: true, redundant: true});
 }
 
+function RebootInitializer(target: {}, propertyKey: string, parameterIndex: number) {
+  set(target.constructor, propertyKey, parameterIndex, {type: MappingType.Direct, initializer: true, reboot: true});
```

Comment:
This looks ok to me but @piotrswigon should review/approve, I only loosely follow the purpose of `initializer` in this context.